docs/developer/new-contributor-projects.html

```
@@ -1697,18 +1696,18 @@ <h2 id="SideEffectsOnly">More precise side effect annotations</h2>
 </p>
 
 
-<h2 id="unrefinement-warnings">Better error messages that are due to side effects</h2>
+<h2 id="unrefinement-warnings">Better error messages for errors that are due to side effects</h2>
 
 <p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 1701.

(tag-pair)